tests/unit/core/reducers/test_addons.js

```diff
@@ -101,6 +101,21 @@ describe(__filename, () => {
     });
   });
 
+  it('stores all add-on slugs in lowercase', () => {
+    const addonResults = [
```

Comment:
Nit: You could avoid hardcoding these by making `FIRST` and `SeCond` consts and then using `toLowerCase()` in the assertion.